cxx4/ncFile.cpp

```diff
@@ -175,3 +175,17 @@ void NcFile::redef(){
 void NcFile::enddef() {
     ncCheck(nc_enddef(myId),__FILE__,__LINE__);
 }
+
+string NcFile::getPath() const
+{
+    size_t pathLength;
+    nc_inq_path(myId, &pathLength, nullptr);
+
+    char *pathCString = new char[pathLength + 1];
+    nc_inq_path(myId, nullptr, pathCString);
+
+    string path (pathCString);
+    delete[] pathCString;
+
+    return path;

```

Comment:
We can avoid using `new/delete` with `std::vector`:

```suggestion
    std::vector<char> pathCString(pathLength + 1);
    nc_inq_path(myId, nullptr, pathCString.data());

    return std::string{pathCString.data()};
```

Reply:
I like it.

What's the word on standard compliance for this project? Extended initializer lists started with C++11.

Reply:
It's an open issue: https://github.com/Unidata/netcdf-cxx4/issues/123

Reply:
Turns out nullptr also came with C++11. How much attention does this repo get?